doc/user/content/get-started/quickstart.md

```diff
@@ -197,306 +92,216 @@ For this guide, you'll use a [built-in load generator](/sql/create-source/load-g
      id | buyer | auction_id | amount |          bid_time
     ----+-------+------------+--------+----------------------------
      10 |  3844 |          1 |     59 | 2023-06-21 21:24:54.838+00
-    (1 row)
     ```
 
-## Step 4. Compute real-time results
+## Step 2.
 
-With auction data streaming in, you can now explore the unique value of Materialize: computing real-time results over fast-changing data.
+To identify potential auction flippers, you need to keep track of the winning
+bids (or, the bid with the highest `amount`) for each completed auction.
 
-1. Switch to your `compute_qck` cluster:
+1. ...
 
     ```sql
-    SET CLUSTER = compute_qck;
+    CREATE VIEW winning_bids AS
+    SELECT DISTINCT ON (auctions.id) bids.*, auctions.seller
+    FROM auctions, bids
+    WHERE auctions.id = bids.auction_id
+      AND bids.bid_time < auctions.end_time
+      AND mz_now() >= auctions.end_time
+    ORDER BY auctions.id, bids.amount DESC;
     ```
 
-1. Enable `psql`'s timing feature so you can see how quickly results are returned:
+    - Note on DISTINCT ON vs ROW_NUMBER(), and why we use it here (it's still a
+      little exotic even for me, so I expect users to be confused here)?
```

Comment:
I think you lean into the weirdness. "Hey, you might not know this but it sure as 💩 works in postgres and so it'll work here too!"